package.json

```diff
@@ -4,7 +4,7 @@
   "description": "GraphicaL User Interface for creating and running Scratch 3.0 projects",
   "main": "./src/index.js",
   "scripts": {
-    "build": "npm run clean && webpack --progress --colors --bail",
```

Comment:
I think I'd rather fix the issue using something like [rimraf](https://github.com/isaacs/rimraf) than remove the clean step for Windows users.